sql/ddl.mdx

```diff
@@ -52,14 +52,41 @@ a Kafka connection called my_kafka and a Kafka topic order_topic:
 ```sql
 CREATE TABLE orders (
   customer_id INT,
-  order_id INT)
- WITH (
+  order_id INT,
+  date_string TEXT,
+  event_time TIMESTAMP GENERATED ALWAYS AS (CAST(date_string as TIMESTAMP)),
+  watermark TIMESTAMP GENERATED ALWAYS AS (event_time - INTERVAL '15' SECOND)
+) WITH (
   connection = 'my_kafka',
   topic = 'order_topic',
-  format = 'json'
+  format = 'json_schema_registry',
+  event_time_field = 'event_time',
+  watermark_field = 'watermark'
 );
 ```
 
+#### CREATE TABLE (Connection) Options
+Connection tables are the most powerful of the table types,
+allowing users to configure a number of options
+that specify and modify the behavior.
+They are specified via the `WITH` clause,
+with an unquoted key and a single or double-quoted value.
+The following options are supported:
+| Option            | Description |
+|--------           |-------------|
+| `connection`      | The name of the connection to use.|
+| `topic`           | The name of the topic to read from or write to. Specific to Kafka connections.|
```

Comment:
I would remove this, as the connector-specific options are documented on the connector pages. It's weird to document just one here.